.github/workflows/ci.yml

```diff
@@ -28,8 +28,10 @@ on:
         required: false
 env:
   FAIL_FAST: ${{ github.event_name == 'pull_request' }}
+  BUILD_JAVA_VERSION: 8 # minimum version
+  TEST_JAVA_VERSION: 17 # preferred version
   MAVEN_OPTS: -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false -Dmaven.wagon.http.retryHandler.class=standard -Dmaven.wagon.http.retryHandler.count=3
-  OZONE_WITH_COVERAGE: ${{ github.repository == 'apache/ozone' && github.event_name == 'push' }}
+  OZONE_WITH_COVERAGE: ${{ github.event_name == 'push' }}
```

Comment:
Does this mean that all forks will now have coverage enabled as well?

Is this intended to be included in the final PR?

Reply:
Coverage will run but publish to Sonar will not per [this line](https://github.com/apache/ozone/blob/master/.github/workflows/ci.yml#L609). Forks would anyways not have the `SONARCLOUD_TOKEN` secret necessary to publish to sonar.

Reply:
Got it. Thanks @errose28 !